source/Core/Field.php

```diff
@@ -80,16 +77,7 @@ public function __construct($value = null, $type = self::T_TEXT)
      */
     public function __isset($name)
     {
-        switch ($name) {
-            case 'rawValue':
-                return ($this->rawValue !== null);
-                break;
-            case 'value':
-                return ($this->value !== null);
-                break;
-            //return true;
-        }
-        return false;
+        return ($this->{$name} !== null);
```

Comment:
Maybe we can get rid of the parentheses here?
```suggestion
        return $this->{$name} !== null;
```